build.fsx

```diff
@@ -9,7 +9,22 @@ open Fake
 open Fake.DotNetCli
 
 // Variables
-let configuration = "Release"
+let product = "Akka.NET"
+let authors = [ "Akka.NET Team" ]
+let copyright = "Copyright © 2013-2017 Akka.NET Team"
+let company = "Akka.NET Team"
+let description = "Akka.NET is a port of the popular Java/Scala framework Akka to .NET"
+let tags = ["akka";"actors";"actor";"model";"Akka";"concurrency"]
+let configuration = if isMono then "Release Mono" else "Release"
```

Comment:
This variable is passed into flags during `dotnet build` and `dotnet test` later in the build script.  I think the only options are `Release` and `Debug` for the `-c` flag on those commands.  `Release Mono` would probably break those commands.